main.py

```diff
@@ -0,0 +1,29 @@
+def tokenize(text):
```

Comment:
what is it? You must implement functions inside lab_1.../main.py file. Not here

Reply:
I think that it's done now :)